drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -227,6 +234,7 @@ def studio_view(self, context):
             'js_templates': js_templates,
             'help_texts': help_texts,
             'field_values': field_values,
```

Comment:
It seems inconsistent to pass the help text and field values as context vars, but to use `self.fields.foo.display_name` to get the display name. Maybe we should just pass `fields` as a context variable and then use `fields.foo.display_name`, `fields.food.help`, and `fields.foo.values`? That would be more consistent and efficient.